tests/ui_tools/test_boxes.py

```diff
@@ -104,35 +104,38 @@ def test_autocomplete_mentions(self, write_box, users_fixture,
             text, state, prefix_string)
         assert typeahead_string == required_typeahead
 
-    @pytest.mark.parametrize('text, state, required_typeahead', [
-        ('#Stream', 0, '#**Stream 1**'),
-        ('#Stream', 1, '#**Stream 2**'),
-        ('#S', 0, '#**Some general stream**'),
-        ('#S', 1, '#**Secret stream**'),
-        ('#S', 2, '#**Stream 1**'),
-        ('#S', 3, '#**Stream 2**'),
-        ('#S', -1, '#**Stream 2**'),
-        ('#S', -2, '#**Stream 1**'),
-        ('#S', -3, '#**Secret stream**'),
-        ('#S', -4, '#**Some general stream**'),
-        ('#S', -5, None),
-        ('#So', 0, '#**Some general stream**'),
-        ('#So', 1, None),
-        ('#Se', 0, '#**Secret stream**'),
-        ('#Se', 1, None),
-        ('#St', 0, '#**Stream 1**'),
-        ('#St', 1, '#**Stream 2**'),
-        ('#Stream 1', 0, '#**Stream 1**'),
-        ('No match', 0, None),
-        ('No match', -1, None)
+    @pytest.mark.parametrize('text, state, prefix_string, \
+                             required_typeahead', [
+        ('#Stream', 0, '#', '#**Stream 1**'),
+        ('#Stream', 1, '#', '#**Stream 2**'),
+        ('#S', 0, '#', '#**Some general stream**'),
+        ('#S', 1, '#', '#**Secret stream**'),
+        ('#S', 2, '#', '#**Stream 1**'),
+        ('#S', 3, '#', '#**Stream 2**'),
+        ('#S', -1, '#', '#**Stream 2**'),
+        ('#S', -2, '#', '#**Stream 1**'),
+        ('#S', -3, '#', '#**Secret stream**'),
+        ('#S', -4, '#', '#**Some general stream**'),
+        ('#S', -5, '#', None),
+        ('#So', 0, '#', '#**Some general stream**'),
+        ('#So', 1, '#', None),
+        ('#Se', 0, '#', '#**Secret stream**'),
+        ('#Se', 1, '#', None),
+        ('#St', 0, '#', '#**Stream 1**'),
+        ('#St', 1, '#', '#**Stream 2**'),
+        ('#Stream 1', 0, '#', '#**Stream 1**'),
+        ('No match', 0, '#', None),
+        ('No match', -1, '#', None)
```

Comment:
@preetmishra  It would be better to pass the prefix string in a separate `parameterize` block, since the value is the same for all the fixtures.

Reply:
@kaustubh-nair Indeed. Thanks for pointing that out. :)